### DISCUSSION TITLE
QList tag

### DISCUSSION THREAD

**Discussion by tomukasas** [quasarframework/quasar#14710] [Ideas / Proposals] [answered]
I think it would be useful to allow other html tags to be used with QList component. It is possible to do that with QItem component, so would make sense to support that on QList component too.

As an example it would be useful to set QList tag to `ul` or `ol` and QItem tag to `li`.

**Comment by pdanpdan** [comment]
Can you please open an issue (feature request) for adding `tag` prop for `QList`?

**Reply by tomukasas** [reply]
Sure thing :) what is the best template to use for creating feature requests?

**Comment by rstoenescu** [comment, marked as answer]
Will be available in Quasar v2.10.1 & v1.22.1

**Reply by tomukasas** [reply]
thanks @rstoenescu
